spec/Section 5 -- Validation.md

```diff
@@ -556,8 +556,15 @@ fragment directFieldSelectionOnUnion on CatOrDog {
 
 FieldsInSetCanMerge(set):
 
-- Let {fieldsForName} be the set of selections with a given _response name_ in
-  {set} including visiting fragments and inline fragments.
+- Let {visitedSelections} be the selections in {set} including visiting fields,
+  fragment-spreads and inline fragments.
```

Comment:
If we visit fields, we'll get the entire tree. We only want to visit the fragment spreads/inline fragments.

Reply:
Fixed in my PR below